site/en/spec/glossary.md

```diff
@@ -329,9 +329,7 @@ Central software component of the controller, executing workflows, including con
 
 ### Processor
 
-A processor is a method provided by a tool that implements the uniform [OCR-D
-CLI](https://ocr-d.de/en/spec/cli) for run-time data processing, and yields one or more
-[activities](#activities).
+A processor is an [OCR-D compliant command-line-interface](https://ocr-d.de/en/spec/cli) for executing a single [workflow step](#activities) on the workspace (represented by local METS). It reads input files for all or requested physical pages of the input fileGrp(s), and writes output files for them into the output fileGrp(s). It may take a number of optional or mandatory parameters.
```

Comment:
The old docstring in `ocrd.processor.base` was not the same as the above glossary item, though. And there's a connection to the similar `evaluator` term entry. (We'll most likely have multiple CLI standards for different tasks, plus non-standard CLIs.) And (as recent developments highlighted) we should also allow for the possibility of multi-task processors. How about the following synthesis:
```suggestion
A processor is a tool that implements the uniform [OCR-D command-line-interface](https://ocr-d.de/en/spec/cli) for run-time data processing. That is, it executes a single [workflow step](#activities), or a combination of multiple workflow steps, on the [workspace](https://ocr-d.de/en/user_guide#preparing-a-workspace) (represented by local [METS](https://ocr-d.de/en/spec/mets)), reading input files for all or requested physical pages of the input fileGrp(s), and writing output files for them into the output fileGrp(s). It may take a number of optional or mandatory [parameters](https://ocr-d.de/en/spec/ocrd_tool).
```